.github/workflows/release.yml

```diff
@@ -43,6 +44,31 @@ defaults:
     shell: bash --noprofile --norc -xeuo pipefail {0}
 
 jobs:
+  determine-run-id:
+    runs-on: ubuntu-latest
+    outputs:
+      run-id: ${{ steps.lookup-run-id.outputs.run-id }}
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8  # v5.0.0
+        with:
+          fetch-depth: 0
```

Comment:
Please add a comment here emphasizing that it should be 0 -- the new `lookup-run-id` script relies on this so that all git tags are fetched!